### ISSUE TITLE
when set clusters for subscribe not trigger callback

### ISSUE THREAD

**Issue by tureo** [nacos-group/nacos-sdk-go#761]
### What version of nacos-sdk-go are you using?
v2.2.6
### What version of nacos-sever are you using?
v2.0.2
### What version of Go are you using (`go version`)?
go version go1.22.2 linux/amd64
### What operating system (Linux, Windows, …) and version?
linux,uos
### What did you do?
If possible, provide a recipe for reproducing the error.
when set clusters for subscribe callback not trigger
https://github.com/nacos-group/nacos-sdk-go/blob/61ecaa81639f4a999681c08cfba416ac6800051e/example/service/main.go#L149
	//Subscribe key=serviceName+groupName+cluster
	//Note:We call add multiple SubscribeCallback with the same key.
	subscribeParam := &vo.SubscribeParam{
		ServiceName: "demo.go",
		GroupName:   "group-a",
		Clusters:    []string{"cluster-a"}, // this set clusters bug
		SubscribeCallback: func(services []model.Instance, err error) {
			fmt.Printf("callback return services:%s \n\n", util.ToJsonString(services))
		},
	}
	subscribe(client, subscribeParam)
### What did you expect to see?
callback trigger
### What did you see instead?
callback not trigger

**Comment by binbin0325** [comment]
Check if cluster-A cluster exists in nacos server

**Comment by youzhixiaomutou** [comment]
![image](https://github.com/user-attachments/assets/df85f5ad-9d97-4d05-886c-b4f1da852985)
![image](https://github.com/user-attachments/assets/a528106d-bd13-47d1-8398-5685a451f5a1)
clusters in notify reply is empty.

**Comment by youzhixiaomutou** [comment]
@tureo It works well with nacos server v2.4.1. But not work with v2.0.2 and v2.1.0.

**Comment by tureo** [comment]
> @tureo It works well with nacos server v2.4.1. But not work with v2.0.2 and v2.1.0.

Thank you, our Nacos server version cannot be upgraded, so our temporary solution is to temporarily not set the cluster value

**Comment by tureo** [comment]
> Check if cluster-A cluster exists in nacos server

@binbin0325  Yes, I am certain that cluster-a exists in Nacos, and here are some [details](https://github.com/nacos-group/nacos-sdk-go/issues/761#issuecomment-2327842695)  and [solutions](https://github.com/nacos-group/nacos-sdk-go/issues/761#issuecomment-2328008027)  for reproducing it
